frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -100,6 +135,23 @@ export const createBuildConfig = (
       break;
   }
 
+  let webhookTriggerData;
+  if (gitType === GitTypes.unsure) {
+    webhookTriggerData = {
+      type: 'Generic',
+      generic: {
+        secretReference: { name: `${name}-webhook-secret` },
```

Comment:
In 3.11 the name is generated and it looks like no Secret is actually created. What you're doing makes sense to me but did you try what 3.11 was doing?

Reply:
I followed the latest 4.1 docs from https://docs.openshift.com/container-platform/4.1/builds/triggering-builds-build-hooks.html. But i did try the 3.11 approach just now and it works. Not sure which one is better though. 3.11 simplifies things a lot.

Reply:
@spadgett Can you suggest which approach we should choose here? One is creating a `WebhookSecret` resource and referencing that in the `BuildConfig` like -
```
type: "GitHub"
github:
  secretReference:
    name: "mysecret"
```
Vs just creating a `uuid` secret on the fly and using it directly like
```
type: "GitHub"
github:
  secret: "mysecret-uuid-string"
```

Reply:
We definitely want to create a webhook secret. Otherwise users with the `view` role will be able to see the secret value in the BuildConfig YAML.